rclcpp/include/rclcpp/logger.hpp

```diff
@@ -89,6 +98,25 @@ RCLCPP_PUBLIC
 rcpputils::fs::path
```

Comment:
I think we should also deprecate this signature, as the eventual goal is to remove `rcpputils::fs::path`.

Reply:
https://github.com/ros2/rclcpp/pull/2579/commits/d5912794c24c1fe8eeee4b4918c900fd147d660a